### PR TITLE
test: add a test for unsupported viewports

### PR DESCRIPTION
**Pull request by OrKoN**
Follow-up to https://github.com/GoogleChromeLabs/chromium-bidi/pull/817

### CODE REVIEW

File: tests/test_viewport.py [hunk 1]
```diff
@@ -53,3 +55,27 @@ async def test_set_viewport(websocket, context_id):
...
+
+
+    with pytest.raises(Exception) as exception_info:
+      await execute_command(
+          websocket, {
+              "method": "browsingContext.setViewport",
+              "params": {
+                  "context": context_id,
+                  "viewport": {
+                      "width": 10000001,
```
Comment (Lightning00Blade) [inline tests/test_viewport.py:72]: Can we have a link to the code that the max value is set. Atm the number seems to come from thin air.

**Review by Lightning00Blade — APPROVED** [review]

File: tests/test_viewport.py (hunk 1 above)
Comment (OrKoN) [reply tests/test_viewport.py:72]: There is a link to the code where the error is thrown.

Comment (OrKoN) [reply tests/test_viewport.py]: Updated

File: src/bidiMapper/domains/context/browsingContextImpl.ts [hunk 2]
```diff
@@ -619,16 +619,30 @@ export class BrowsingContextImpl {
...
+      } catch (err) {
+        if (
+          (err as Error).message.startsWith(
+            // https://crsrc.org/c/content/browser/devtools/protocol/emulation_handler.cc;l=257;drc=2f6eee84cf98d4227e7c41718dd71b82f26d90ff
+            'Width and height values must be positive'
+          )
+        ) {
+          throw new Message.UnsupportedOperationException(
+            'Provided viewport dimensions are not supported'
+          );
```
Comment (thiagowfx) [inline src/bidiMapper/domains/context/browsingContextImpl.ts:642]: I would print the provided dimensions in the error message as well

**Pull request merged**